store/sqlstore/webhook_store.go

```diff
@@ -35,6 +35,8 @@ func newSqlWebhookStore(sqlStore SqlStore, metrics einterfaces.MetricsInterface)
 		table.ColMap("TeamId").SetMaxSize(26)
 		table.ColMap("DisplayName").SetMaxSize(64)
 		table.ColMap("Description").SetMaxSize(500)
+		table.ColMap("Username").SetMaxSize(255)
+		table.ColMap("IconURL").SetMaxSize(1024)
```

Comment:
Why 1024? What is the length of this field currently? Will 1024 be enough?

Reply:
Well right now that field in MySQL is 255 characters and in Postgres is a text field. The problematic situation could be in Postgres. We can remove the `MaxiSize` and that would convert those fields in text ones.

Reply:
I think we should do that. My worry is this breaks the upgrade for customers if they have a long one already in the database with Postgres.

Reply:
@grundleborg 

I've found that we're enforcing 1024 for IconURL in our webhook models, both in the [incoming](https://github.com/mattermost/mattermost-server/blob/master/model/incoming_webhook.go#L101-L107) and the [outgoing](https://github.com/mattermost/mattermost-server/blob/master/model/outgoing_webhook.go#L189-L195), so no one should have an IconURL larger than 1024 characters. It also checks that the Username is smaller than 65 characters.

So I think is not needed to include a text field. What do you think?

Reply:
Yeah in that case I'm fine with it like this.